samples/Standalone.MvcSample/Controllers/AccountController.cs

```diff
@@ -50,6 +51,11 @@ public IActionResult ExternalLogin(string provider)
                 }
             };
 
+            if (provider.Equals(BankIdAuthenticationDefaults.SameDeviceAuthenticationScheme))
+            {
+                props.Items["cancelReturnUrl"] = "~/";
```

Comment:
Actually I don't think we need it in the IdentityServer sample either. This was a leftover from a previous implementation where both samples needed it. The only reason I can think of on why we would override is if we want cancel to go back "home" after using PIN instead of QR in the same device scheme. 

Perhaps it is enough to put in a sample in the documentation?

Reply:
You are correct of course!